.github/workflows/publish-on-pypi.yml

```diff
@@ -71,24 +103,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -e .[all]
-        pip install -r .github/workflows/requirements-docs.txt --upgrade
-
-    - name: Generate changelog
-      uses: charmixer/auto-changelog-action@v1
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Commit files
-      run: |
-        git config --local user.email "dev@optimade.org"
-        git config --local user.name "optimade_devs"
-        mv CHANGELOG.md docs/
-        git add docs/CHANGELOG.md && git commit -m 'Updated CHANGELOG.md' && echo ::set-env name=push::1 || echo "No changes to CHANGELOG.md"
-
-    - name: Push changes
-      uses: ad-m/github-push-action@master
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+        pip install -r requirements-docs.txt --upgrade
```

Comment:
As with the comment above, is `--upgrade` really needed here together with a requirements.txt?